lib/compiler_rt/stack_probe.zig

```diff
@@ -17,7 +17,7 @@ comptime {
         // Default stack-probe functions emitted by LLVM
         if (is_mingw) {
             @export(_chkstk, .{ .name = "_alloca", .linkage = strong_linkage });
-            @export(___chkstk_ms, .{ .name = "___chkstk_ms", .linkage = strong_linkage });
+            @export(___chkstk_ms, .{ .name = "___chkstk_ms", .linkage = linkage });
```

Comment:
I cannot remember why we ended up with `strong_linkage` here but I would imagine we should replace all instances in this file with `linkage` instead. My suggestion is to do just that in this patch and let's see if anything breaks in the CI.

Reply:
Sounds good, done.